descwl_shear_sims/simple_sim.py

```diff
@@ -262,13 +266,18 @@ def __init__(
         self.bad_columns = bad_columns
         self.bad_columns_kws = bad_columns_kws or {}
 
-        # the SE image could be rotated, so we make it big enough to cover the
-        # whole coadd region plus we make sure it is odd
-        self.se_dim = (
-            int(np.ceil(self.coadd_dim * np.sqrt(2))) + 10 + 2*edge_width
-        )
-        if self.se_dim % 2 == 0:
-            self.se_dim = self.se_dim + 1
+        if se_dim is not None:
+            self.se_dim = int(se_dim)
+            assert self.se_dim > 2*edge_width
```

Comment:
Shouldn't this lower limit be bigger?

Reply:
I guess we could add an arbitrary number to it

Reply:
Well we have the coadd dim and we might have rotations, so I think the default is the min.

Reply:
But the point of the se_dim keyword is so the user can set it to whatever size they want, even something that could have an edge in it and get rejected later

Reply:
But this keyword is set for all epochs. Why would we want to reject all of them?

Reply:
good point